concepticondata/references/references.bib

```diff
@@ -1,3 +1,13 @@
+@article{Ferre2022,
+  title = {Love me in {{L1}}, but hate me in {{L2}}: {{How}} native speakers and bilinguals rate the affectivity of words when feeling or thinking about them},
+  author = {Ferr{\'e}, Pilar and Guasch, Marc and {Stadthagen-Gonzalez}, Hans and Comesa{\~n}a, Montserrat},
+  year = {2022},
+  volume = {3},
+  journal = {Bilingualism: Language and Cognition},
+  pages = {1-15},
```

Comment:
I get the following information for volume and pages from the journal website:

| Bilingualism: Language and Cognition, 25(5), 786-800

So this should be adapted for number, issue, and pages

Reply:
Thanks! I downloaded the PDF a while ago so they must have updated it.